packages/pages/src/dev/server/autoInit.ts

```diff
@@ -65,7 +65,7 @@ const logErrorAndExit = (error: string | any) => {
 
 const validUniverses = ["sandbox", "production", "sbx", "prod", "qa", "dev"];
 
-const parseYextrcContents = () => {
+export const parseYextrcContents = () => {
```

Comment:
Can you move this to its own util file/function too? It's used in two places now and 3 with Alexis' PR.